### PR TITLE
N27: graph game

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #818_

### CODE REVIEW

**Comment by Vrixyz** [comment]
I’ll clean up the lints when I can 🤞

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
The content looks good, but I think you've put the section in the table of contents rather than under the actual 'Game Updates' heading - could you move it down?

**Comment by Vrixyz** [comment]
Thanks, should be fixed now 🙏🏼

File: content/news/027/index.md [hunk 1]
```diff
@@ -84,6 +84,21 @@ small update to improve the UI and fix a few bugs.
...
+The [Graph Game](https://github.com/Vrixyz/graph_nav) uses
+[bevy engine](https://bevyengine.org/), you can play it from
+[browser](https://vrixyz.github.io/graph_nav/) 🕹️, click on colored triangles,
+guess the rules and survive as long as possible!
+
+Be warned, the development just began,
+and the future of the project is not clear,
+feel free to come discuss 💬 with the developer on
+[Discord](https://discord.gg/ZeRkj8pD4n).
+
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:101]:
```suggestion
### [Graph Game]

![image/gameplay of the game: circle and triangles](graph_game.gif)
_Navigating the graph map can be stressful_

[Graph Game] ([GitHub][graph-game-github])
uses [Bevy](https://bevyengine.org/) as its engine. You can play it from your
[browser](https://vrixyz.github.io/graph_nav/) - click on colored triangles,
guess the rules and survive as long as possible!

Development has just begun, and the future of the project is not clear -
the developer welcomes you to come and discuss next steps on the game's
[Discord server][graph-game-discord].

[Graph Game]: https://vrixyz.github.io/graph_nav/
[graph-game-github]: https://github.com/Vrixyz/graph_nav
[graph-game-discord]: https://discord.gg/ZeRkj8pD4n
```

**Review by 17cupsofcoffee — APPROVED** [review]
🚧 Just making some tweaks to match the style of other sections, looks good though!

File: content/news/027/index.md [hunk 2]
```diff
@@ -84,6 +84,23 @@ small update to improve the UI and fix a few bugs.
...
+guess the rules and survive as long as possible!
+
+Development has just begun, and the future of the project is not clear -
+the developer welcomes you to come and discuss next steps on the game's
+[Discord server][graph-game-discord].
+
+[Graph Game]: https://vrixyz.github.io/graph_nav/
+[graph-game-github]: https://github.com/Vrixyz/graph_nav
+[graph-game-discord]: https://discord.gg/ZeRkj8pD4n
 ## Engine Updates
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:104]:
```suggestion
[graph-game-discord]: https://discord.gg/ZeRkj8pD4n

## Engine Updates
```

**Pull request merged**